docs/cluster-overview.md

```diff
@@ -45,7 +45,7 @@ There are several useful things to note about this architecture:
 
 # Cluster Manager Types
 
-The system currently supports three cluster managers:
+The system currently supports four cluster managers:
```

Comment:
Heh OK. Let's just say 'several' to reduce future maintenance. Anything else that can be fixed in this doc?

Reply:
ha good idea, I'll do that.

Nope everything else was great!

Reply:
fixed